bool update
---
bool is_tracking